data/tlg0007/tlg071/__cts__.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0007" projid="greekLit:tlg071" urn="urn:cts:greekLit:tlg0007.tlg071" xml:lang="grc">
+    <ti:title xml:lang="lat">Quomodo quis suos in virtute sentiat profectus</ti:title>
+    
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg071.perseus-eng3" workUrn="urn:cts:greekLit:tlg0007.tlg071" xml:lang="eng">
+        <ti:label xml:lang="eng">How a Man May Become Aware of His Progress in Virtue</ti:label>
+        <ti:description xml:lang="mul">Plutarch. Moralia, Vol. I. Babbitt, Frank Cole, translator. Cambridge, MA: Harvard University Press. London: William Heinemann Ltd. 1927 (printing).</ti:description>
+    </ti:translation>
+    
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg071.perseus-eng4" workUrn="urn:cts:greekLit:tlg0007.tlg071" xml:lang="eng">
+        <ti:label xml:lang="eng">How a Man May Be Sensible of His Progress in Virtue.</ti:label>
+        <ti:description xml:lang="eng">Plutarch. Plutarch's Morals, Vol. II. Goodwin, William W., editor; Ford, Simon, translator. Boston: Little, Brown, and Company. Cambridge. 1874.</ti:description>
```

Comment:
The translator appears to be Mr. Tod: http://www.archive.org/stream/plutarchsmoralst02plutuoft#page/xi/mode/1up
DNB lists him as Hugh Todd.

So instead of `Ford, Simon, translator.` it should read `Tod(d), Hugh, translator.`